build.sh

```diff
@@ -13,11 +13,18 @@ shift "$(( OPTIND - 1 ))"
 NAME=$(echo "${PWD##*/}" | tr _ -)
 TAG="latest"
 
+UID="$(id -u "${USER}")"
+GID="$(id -g "${USER}")"
+
 if [ "$REBUILD" -eq 1 ]; then
     docker build \
```

Comment:
I like the approach with multistage but shouldn't you add an argument 'target' now in the build command?

Reply:
probably but I guess it depends more on the usage. For my own usage I need to build till the end. I guess for you, you are more interested in building only the first part and adding eventually another layer. If you don't mind I let you do it as I have no idea how to use this further.